router/worker.go

```diff
@@ -308,13 +310,20 @@ func (w *worker) transform(routerJobs []types.RouterJobT) []types.DestinationJob
 			w.rt.logger.Debugn("traceParent is empty during router transform", logger.NewIntField("jobId", job.JobMetadata.JobID))
 		}
 	}
-	w.rt.routerTransformInputCountStat.Count(len(routerJobs))
-	destinationJobs := w.rt.transformer.Transform(
-		transformer.ROUTER_TRANSFORM,
-		&types.TransformMessageT{Data: routerJobs, DestType: strings.ToLower(w.rt.destType)},
-	)
-	w.rt.routerTransformOutputCountStat.Count(len(destinationJobs))
-	w.recordStatsForFailedTransforms("routerTransform", destinationJobs)
+	destinationJobs := make([]types.DestinationJobT, 0)
```

Comment:
```suggestion
	destinationJobs := make([]types.DestinationJobT, 0, len(routerJobs))
```

Reply:
output jobs and input jobs need not be the same as there will be batching logic